lib/3rd_party_adapters/LVGL/Screen.cpp

```diff
@@ -0,0 +1,472 @@
+#include "Screen.hpp"
+#include <math.h>
+#include <stack>
+
+static std::stack<IScreen *> screenHistory;
```

Comment:
I do not like the memory management: `IScreen`s are create with `new`. Those are added by a member function to a stack. Leaving a screen leads to `delete`ing. The `new`...`delete` chain is not obvious. 

```suggestion
static std::stack<std::unique_ptr<IScreen *>> screenHistory;
```

Reply:
I will implment it as sugested, but we also discussed that the "delete" must be removed.

Reply:
The pointers were changed to shared_ptr, as unique_ptr lose their existens as soon as the go out of scope.

They might be "std::move"ed into the list container, but I didn't try